index.html

```diff
@@ -11,55 +11,56 @@
     <div class="jumbotron">
       <div class="container">
         <img src="/assets/images/trucol-logo.svg" alt="TruCol logo">
-        <p><i>TruCol</i> is a free and open-source reference guide that explains how to use Markdown, the simple and easy-to-use markup language you can use to format virtually any document.</p>
+        <p><i>TruCol</i> is a decentralised collaboration protocol, which promotes <em>inclusive</em>, <em>fair</em> and <em>accessible</em> work, by
+          enabling developers to participate in the market regardless of their circumstances.</p>
         <p><a class="btn btn-outline-inverse btn-lg" href="/getting-started/" role="button">Get Started</a></p>
       </div>
     </div>
     {% if jekyll.environment == "production" %}<center><div class="horizontal" data-ea-publisher="markdownguide" data-ea-type="image"></div></center>{% endif %}
     <div class="container">
       <div class="md-guide-feature">
-        <h2 class="no-anchor">Everything you need to learn Markdown.</h2>
-        <p class="lead" style="margin-bottom: 60px">Markdown makes writing on the web fast and easy. <em>The Markdown Guide</em> teaches you how to use it.</p>
+        <h2 class="no-anchor">Trucol, decentralising test driven development.</h2>
+        <p class="lead" style="margin-bottom: 60px">The TruCol protocol presents an improvement of market efficiency and developer autonomy by decentralisation and automation of test-driven development.</p>
```

Comment:
I would remove `and automation` as we are not yet able to substitute test-driven development work with AI-engines.

Reply:
I have removed the ```and automation``` part.